### ISSUE TITLE
agent/nodes.py cannot parse f-string when it's numeric

### ISSUE THREAD

**Issue by nicolefinnie** [mkdocstrings/griffe#80]
**Describe the bug**

When `pyupgrade` automatically upgrades our `format`ted string to a f-string, `griffe.agents.nodes.py` cannot parse the f-string and throws an error 

```sh
  File "/home/nicolefinnie/venv/myproject/lib/python3.8/site-packages/griffe/agents/nodes.py", line 764, in _get_call_value
    posargs = ", ".join(get_value(arg) for arg in node.args)
```

**To Reproduce**

Run `mkdocs serve` for this code snippet

```python
key = 0.02
a = f"{round(key, 2)}"
```

**Expected behavior**
no error should be thrown and _get_call_value() should be able to accept a numeric value.

**System (please complete the following information):**
- `griffe` version: 0.14.0
- Python version: 3.8.5
- OS: Ubuntu 20.4 LTS

**Suggested fix**

casting the value to `str` in the file agents/nodes.py, line 764, however, I am not sure about the side effect, which has to be validated through the CI/CD pipeline.
```python

def _get_call_value(node: NodeCall) -> str:
    # cast get_value() to string
    posargs = ", ".join(str(get_value(arg)) for arg in node.args)
    kwargs = ", ".join(get_value(kwarg) for kwarg in node.keywords)
    if posargs and kwargs:
        args = f"{posargs}, {kwargs}"
    elif posargs:
        args = posargs
    elif kwargs:
        args = kwargs
    else:
        args = ""
    return f"{get_value(node.func)}({args})"

```

**Comment by pawamoy** [comment]
Thanks, should be fixed now, in 0.20.0 :slightly_smiling_face:

**Comment by nicolefinnie** [comment]
> Thanks, should be fixed now, in 0.20.0 slightly_smiling_face

Awesome thanks!

**Comment by wsascha** [comment]
Wow, that was fast! Thanks for the fix!
